### ISSUE TITLE
key/fqfield error when requesting restricted field

### ISSUE THREAD

**Issue by bastianjoel** [OpenSlides/openslides-autoupdate-service#789]
When requesting the following payload without an authentication token set an error is sent which does not seem right. 

Request: 
```json
[
  {
    "collection": "user",
    "ids": [
      1
    ],
    "fields": {
      "default_password": null,
      "can_change_own_password": null,
      "title": null,
      "first_name": null,
      "last_name": null,
      "pronoun": null,
      "username": null,
      "gender": null,
      "default_number": null,
      "default_structure_level": null,
      "default_vote_weight": null,
      "is_physical_person": null,
      "is_active": null,
      "meeting_ids": null,
      "saml_id": null,
      "email": null,
      "last_email_sent": null,
      "last_login": null,
      "organization_management_level": null,
      "committee_ids": null,
      "committee_management_ids": null,
      "meeting_user_ids": {
        "type": "relation-list",
        "collection": "meeting_user",
        "fields": {
          "about_me": null,
          "user_id": null,
          "meeting_id": null,
          "group_ids": null,
          "vote_delegated_to_id": null,
          "vote_delegations_from_ids": {
            "type": "relation-list",
            "collection": "meeting_user",
            "fields": {
              "meeting_id": null,
              "group_ids": null,
              "id": null,
              "user_id": {
                "type": "relation",
                "collection": "user",
                "fields": {
                  "title": null,
                  "first_name": null,
                  "last_name": null,
                  "pronoun": null,
                  "username": null,
                  "gender": null,
                  "default_number": null,
                  "default_structure_level": null,
                  "default_vote_weight": null,
                  "meeting_user_ids": null,
                  "id": null
                }
              }
            }
          },
          "vote_weight": null,
          "comment": null,
          "structure_level": null,
          "number": null,
          "id": null
        }
      },
      "is_present_in_meeting_ids": null,
      "id": null
    }
  }
]
```

Error:
```json
{
  "error": {
    "type": "invalid",
    "msg": "the key/fqfield is invalid: user/0/organization_management_level"
  }
}
```

Besides from that the error message is sent with a status code 200. It would be better to receive an authentication error or at least a non ok status code in that case.

**Comment by bastianjoel** [comment]
Sending the correct status is quite important here as the client uses that information to control certain recovery logic. 

When resolving a long polling connection with a status code 200 the client currently assumes that the connection could be reopened immediately after a health check (for example after a reboot of the service).

**Comment by bastianjoel** [comment]
I ran into this issue again today with current main.

**Comment by ostcar** [comment]
> I ran into this issue again today with current main. 

Can you tell me as detailed as possible, how I can reproduce it?

**Comment by bastianjoel** [comment]
I am using the dev setup with the original example data (`make run-dev`). 

When I do the first step as you described it:
```
curl -k https://localhost:8000/system/autoupdate?single=true -d @request
```

I get 
```
{"error": {"type": "invalid", "msg": "the key/fqfield is invalid: user/0/organization_management_level"}}
```

**Comment by jsangmeister** [comment]
I can reproduce this.

**Comment by ostcar** [comment]
I had another look and was able to reproduce it. The problem was the field `user/default_password`. So it was also possible to cause the error with this call: `curl -k 'https://localhost:8000/system/autoupdate?k=user/1/default_password'`.

The reason for that error was, that I checked the DB-Field `user/organization_management_level` with the request user in the restricter for user/modeH. So in this case `user/0/organization_management_level`.

I checked all other places, where I do a DB request for the request user and found another case for the personal_note. I am confident, that I found all places and that the error should not happen again (future changes not included).
